modules/src/ics02_client/query.rs

```diff
@@ -88,19 +88,12 @@ where
         query: QueryClientFullState<CLS>,
         response: AbciQuery,
     ) -> Result<Self, error::Error> {
-        match (response.value, &response.proof) {
-            (Some(value), _) => {
-                let client_state = amino_unmarshal_binary_length_prefixed(&value)?;
-
-                Ok(ClientFullStateResponse::new(
-                    query.client_id,
-                    client_state,
-                    response.proof,
-                    response.height.into(),
-                ))
-            }
-            (None, _) => Err(error::Kind::Rpc.context("Bad response").into()),
-        }
+        Ok(ClientFullStateResponse::new(
+            query.client_id,
+            amino_unmarshal_binary_length_prefixed(&response.value)?,
```

Comment:
Ran this locally and it errored here on type check. This function expects a `&[u8]` but `response.value` is an `Option<vec<u8>>` How come this didn't fail in CI?  🤔

Reply:
still seeing this? `cargo update` maybe? loos like `response.value` still points to the old `Option<vec<u8>>` version of tm.